.pre-commit-config.yaml

```diff
@@ -0,0 +1,59 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+default_install_hook_types: [pre-push]
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-shebang-scripts-are-executable
+      - id: check-merge-conflict
+      # - id: check-symlinks
+      - id: check-toml
+      - id: check-xml
+      - id: check-yaml
+      - id: debug-statements
+      - id: destroyed-symlinks
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: fix-byte-order-marker
+      - id: mixed-line-ending
+      - id: requirements-txt-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/ansible/ansible-lint
+    rev: v25.1.0
```

Comment:
Version should match the ansible-lint workflow, I think? (We can update that one to v25 if you'd like.)

Reply:
Thanks, I missed that. I'll leave v24.10.0, which is the version in Debian testing